conf/airframes/examples/quadshot_asp21_spektrum.xml

```diff
@@ -170,71 +209,161 @@
+    <define name="MAX_ERR_ZD"  value="SPEED_BFP_OF_REAL( 10.)"/>
+    <define name="MAX_SUM_ERR" value="2000000"/>
     <define name="HOVER_KP"    value="200"/>
     <define name="HOVER_KD"    value="175"/>
-    <define name="HOVER_KI"    value="72"/>
-    <define name="NOMINAL_HOVER_THROTTLE" value ="0.4"/>
+    <define name="HOVER_KI"    value="100"/>
+    <define name="NOMINAL_HOVER_THROTTLE" value ="0.2"/>
     <define name="ADAPT_THROTTLE_ENABLED" value="FALSE"/>
   </section>
 
   <section name="AHRS" prefix="AHRS_">
-    <define name="H_X" value=" 0.47577"/>
-    <define name="H_Y" value=" 0.11811"/>
-    <define name="H_Z" value=" 0.87161"/>
+    <!--     Delft magnetic field -->
+    <define name="H_X" value="0.39049610"/>
+    <define name="H_Y" value="0.00278894"/>
+    <define name="H_Z" value="0.92060036"/>
   </section>
 
-  <!-- Gains for horizontal navigation-->
+ <!-- Gains for horizontal navigation-->
   <section name="GUIDANCE_H" prefix="GUIDANCE_H_">
-    <define name="PGAIN" value="100"/>
-    <define name="DGAIN" value="100"/>
-    <define name="IGAIN" value="0"/>
+    <define name="PGAIN" value="50"/>
+    <define name="DGAIN" value="50"/>
+    <define name="IGAIN" value="65"/>
+
+    <define name="MAX_BANK" value="RadOfDeg(55)"/>
+    <define name="FORWARD_MAX_BANK" value="RadOfDeg(35)"/>
+    <define name="USE_REF" value="FALSE"/>
+  </section>
+
+  <section name="NAV">
+    <define name="ARRIVED_AT_WAYPOINT" value="5" unit="m"/>
   </section>
 
   <section name="MISC">
-    <define name="REF_QUAT_INFINITESIMAL_STEP" value="TRUE"/>
+    <define name="RADIO_CONTROL_SPEKTRUM_SIGNS" value="{1,1,1,1,1,1,1,1,1,1,1,1}"/>
     <!--The Quadshot uses a slightly different axis system for the setpoint, to make both hovering and flying forward intuitive-->
     <define name="USE_EARTH_BOUND_RC_SETPOINT" value="TRUE"/>
     <!-- This is the pitch angle that the Quadshot will have in forward flight, where 0 degrees is hover-->
-    <define name="TRANSITION_MAX_OFFSET" value="-82.0" unit="deg"/>
-    <define name="RADIO_CONTROL_SPEKTRUM_SIGNS" value=" {1,1,-1,-1,1,-1,1,1,1,1,1,1}"/>
+    <define name="TRANSITION_MAX_OFFSET" value="-75.0" unit="deg"/>
+
+    <define name="RC_LOST_MODE" value="MODE_AUTO2"/>
+
+    <define name="SWITCH_STICKS_FOR_RATE_CONTROL" value="TRUE"/>
+
+    <define name="AIRSPEED_ETS_I2C_DEV" value="i2c2"/>
+
+    <define name="AHRS_MAG_UPDATE_ALL_AXES" value="FALSE"/>
+
+    <define name="THRESHOLD_GROUND_DETECT" value="25.0"/>
+    <define name="KILL_ON_GROUND_DETECT" value="TRUE"/>
+    <define name="FAILSAFE_GROUND_DETECT" value="TRUE"/>
+    <define name="HYBRID_NAVIGATION" value="TRUE"/>
+    <define name="NO_RC_THRUST_LIMIT" value="TRUE"/>
+    <define name="ADAPTIVE_INDI" value="FALSE"/>
+  </section>
+
+  <section name="SIMULATOR" prefix="NPS_">
+    <define name="ACTUATOR_NAMES"  value="{&quot;front_motor&quot;, &quot;back_motor&quot;, &quot;right_motor&quot;, &quot;left_motor&quot;}"/>
```

Comment:
please use the `type="string[]"` attribute like in the other up-to-date airframe files...